analyzer/consensus/static/accounts.go

```diff
@@ -107,3 +165,27 @@ func QueueConsensusAccountsFirstActivity(batch *storage.QueryBatch, chainName co
 
 	return nil
 }
+
+func QueueConsensusDelegationsSnapshots(batch *storage.QueryBatch, chainName common.ChainName, logger *log.Logger) error {
```

Comment:
Can you please add a delegation/README.md file that explains how the delegation files were obtained